index.html

```diff
@@ -759,10 +759,11 @@ <h3>
         <p>
           The [=manifest's=] <code><dfn data-export="" data-dfn-for=
           "manifest">start_url</dfn></code> member is a <a>string</a> that
-          represents the <dfn>start URL</dfn> , which is <a>URL</a> that the
-          developer would prefer the user agent load when the user launches the
-          web application (e.g., when the user clicks on the icon of the web
-          application from a device's application menu or homescreen).
+          represents the <dfn data-export="">start URL</dfn> , which is
```

Comment:
I'm still a bit worried about exporting "start URL" without an associated thing... could it be:

```suggestion
          represents the <dfn class"export" data-dfn-for="Web Application">start URL</dfn> , which is
```

Reply:
Actually, seems we have defined "installed web application" 🎉

Reply:
This is commenting on a PR that was already submitted (#1112). (It just got caught up in this PR's delta because it wasn't submitted at the time I uploaded this.) I've rebased now, which should make things clearer.

You may be right, but I would like you to put this up as a separate PR if you don't mind. It's a bit complex as I've now got text in [manifest-incubations](https://wicg.github.io/manifest-incubations/#home_tab-member) which links to this and will have to be updated when you make this change.